### ISSUE TITLE
[Feature Request]: Make preview/green to be enabled/disabled

### ISSUE THREAD

**Issue by curiositize** [josephdadams/TallyArbiter#268]
### What feature would you like to see added to TallyArbiter?

I there!

Thanks for all the work!

I have a (maybe) little request. In Germany preview/green is not that common. Also in corporate environments even red tally is hard for some talents.. So I'd like to be able to disable preview/green in TallyArbiter completely and of course enable it depending on the event type.

Hope that others find this request useful as well.

Thanks and take care!
Thomas

**Comment by JTF4** [comment]
Hey @curiositize! I believe that this is coming in V3. We will have customizable colors for each state and you should be able to just change the preview state to black so a light will not turn on.

**Comment by curiositize** [comment]
Hello @JTF4! That sounds awesome! Looking forward to it! Thanks for the quick reply!

**Comment by josephdadams** [comment]
Yes, once we release all the updated listener clients that are compatible with v3.0 and you upgrade everywhere, you'll be able to do this.

**Comment by mg-1999** [comment]
Version 3.0.0 has been released. Could you try it out and let us know if this the behaviour you expect?

**Comment by BenJamesAndo** [comment]
Hi. I'm not the OP but I did test this and changing it to black works great. However if you have text displaying when not on program, then that disappears when it's showing preview/black. E.g. I'm using the M5Atom and have the camera number displayed. When live it goes red, when not live it shows the camera number, but when on preview the camera number disappears. I would think a better implementation would be the ability to disable it entirely, or 'display' a fully transparent colour, thus allowing anything else that's already displaying to remain to do so.

**Comment by josephdadams** [comment]
Resolved in v3.0.0 -- bus colors (including setting Preview to black) are now fully customizable per-device. Closing since this was confirmed working. Please reopen if you're still seeing issues.
